src/com/itacademy/eshop/services/ShopService.java

```diff
@@ -23,7 +27,19 @@ private ArrayList<Product> createProducts() {
          * one per each category in Category enum.
          * Product should not have any reviews.
          */
-        throw new UnsupportedOperationException("createProducts() is not implemented yet");
+
+
+        ArrayList<Product> arrayList = new ArrayList<>();
```

Comment:
Don't use type in the variable name. I suggest to have a plural for the objects you want to have list for. In this case`List<Product> products = new ArrayList<>();`